addusers.py

```diff
@@ -257,6 +246,30 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
+    
+    help_description = ("Add new users and projects to OpenStack using " 
+                        "data from Google Sheets.")
+    parser = argparse.ArgumentParser(description=help_description)
+    parser.add_argument('-c', '--config', 
+                        help='Specify configuration file.')
+
+    args = parser.parse_args()
+   
+    if args.config is not None:
```

Comment:
This conditional is not required. If ``args.config`` is ``None`` you are sending None to ``set_config_file(cfg_file=None)`` which knows how to interpret the ``None`` value.